### DISCUSSION TITLE
Error on CRTBNDRTPG actions - undefined library

### DISCUSSION THREAD

**Discussion by robinsg** [codefori/vscode-ibmi#231] [Q&A]
Hello,

I'm running v0.6.6.

My source file is in the IFS but when I run the CRTBNDRPG I get the following:

**Action Create Bound RPG Program (CRTBNDRPG) for undefined/RPABOTS001 was not successful.**

This is missing the object library name from what I can tell.

If I prompt the CRTBNDRPG action and type in the object library the compile works fine.

In the actions I can see the following for the CRTBDNRPG stream file option:

`CRTBNDRPG PGM(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)`

From the documentation &BUILDLIB is meant to reflect the &CURLIB setting.

When I press the  Change Current Library icon on my User Library List pane nothing seems to happen.

Any suggestion how I correct this issue?

**Comment by worksofliam** [comment]
This is going to sound like a stupid question, but are you connected to a system when you click to Change Current Library button?

You could also try editing the settings.json manually and updating it from there for that connection.

**Reply by robinsg** [reply]
Hahahaha. I'm most definitely connected to a system. I have an IFS file open and I can run actions.

So, how do I set this value in the settings?

**Reply by worksofliam** [reply]
You will want to press `Command+,` or `Control+,` and navigate to the following:

<img width="776" alt="Screen Shot 2021-08-03 at 9 22 02 AM" src="https://user-images.githubusercontent.com/3708366/128022704-c6716d1d-65b8-4503-8a09-5a6bee4540e9.png">

**Comment by worksofliam** [comment]
You are the second person to report this today.

**Comment by worksofliam** [comment]
Moved over to issue instead.
